client/apps/edit/components/admin/index.coffee

```diff
@@ -152,3 +157,42 @@ module.exports = class EditAdmin extends Backbone.View
     id = $(e.currentTarget).attr 'data-id'
     @article['featured' + resource].remove id
     @article.save()
+
+  setupSlug: ->
+    if $('.edit-admin-slug-input').val() is @generateSlugFromTitle()
+      $('.edit-admin-slug-generate').hide()
+    else
+      $('.edit-admin-slug-generate').show()
+
+    $('.edit-admin-slug-input').on 'keyup', _.debounce( =>
+      @slugify()
+      @article.save slug: $('.edit-admin-slug-input').val()
+    , 1000 )
+
+  slugify: ->
+    t = $('.edit-admin-slug-input')
```

Comment:
Nitpick, but I prefer to preface jQuery elements with dollar sign, e.g. `$t = ` for clarity.